code/modules/client/preference/loadout/loadout_suit.dm

```diff
@@ -96,6 +96,11 @@
 	path = /obj/item/clothing/suit/jacket/secbomber
 	allowed_roles = list("Head of Security", "Warden", "Detective", "Security Officer")
 
+/datum/gear/suit/secponcho
+	display_name = "Poncho, security"
+	path = /obj/item/clothing/suit/armor/secponcho
+	allowed_roles = list("Head of Security", "Warden", "Detective", "Security Officer")
+
```

Comment:
better to move this next to the other ponchos

Reply:
You think? I thought its more of a sec gear than poncho

Reply:
The others seem to be grouped by clothing category instead of departments, with this one being in the middle of bomber jackets category.

Reply:
moved it to ponchos